modules/learn/partials/transaction-privileges.adoc

```diff
@@ -0,0 +1,11 @@
+To execute a key-value operation within a transaction, users must have the relevant _Administrative_ or _Data_ RBAC roles, and permissions on the relevant buckets, scopes and collections.
+
+Similarly, to run a query statement within a transaction, users must have the relevant _Administrative_ or _Query & Index_ RBAC roles, and permissions on the relevant buckets, scopes and collections.
+
+Refer to xref:learn:security/roles.adoc[Roles] for details.
+
+[NOTE]
+.Query Mode
+When a transaction executes a query statement, the transaction enters query mode, which means that the query is executed with the user's query permissions.
+Any key-value operations which are executed by the transaction _after_ the query statement are _also_ executed with the user's query permissions.
+These may be different to the user's data permissions, which may lead to unexpected results.
```

Comment:
The last statement should be rephrased.
These may or may not be different from the user's data permissions.

Reply:
Done